tests/syft/lib/python/namedtuple/namedtuple_test.py

```diff
@@ -4,44 +4,25 @@
 # syft absolute
 from syft import serialize
 from syft.core.common.serde.deserialize import _deserialize
-from syft.lib.python import ValuesIndices
 
 
-def test_torch_valuesindices_serde() -> None:
+def test_torch_return_types_serde() -> None:
     x = torch.Tensor([[1, 2], [1, 2]])
```

Comment:
Can we add a test for each type so we are confident they all work?

Reply:
will do